src/components/InstancesList/InstancesList.js

```diff
@@ -1027,7 +1027,7 @@ class InstancesList extends React.Component {
     const pagingCanGoPrevious = browseQueryExecuted ? !!other?.prev : null;
 
     const validateDataQuery = (query) => {
-      const endsWithAsterisk = /\*$/;
+      const endsWithAsterisk = /\*/;
```

Comment:
Please rename the variable to `containsAsterisk` to reflect what it contains

Reply:
Yeah, sorry about that.